plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -689,19 +688,23 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
 
         Database schema = metastore.getDatabase(schemaName).orElseThrow(() -> new SchemaNotFoundException(schemaName));
 
-        boolean external = true;
         String location = getLocation(tableMetadata.getProperties());
-        if (location == null) {
-            String schemaLocation = getSchemaLocation(schema)
-                    .orElseThrow(() -> new TrinoException(NOT_SUPPORTED, "The 'location' property must be specified either for the table or the schema"));
-            String tableNameForLocation = tableName;
-            if (useUniqueTableLocation) {
-                tableNameForLocation += "-" + randomUUID().toString().replace("-", "");
-            }
-            location = new Path(schemaLocation, tableNameForLocation).toString();
-            checkPathContainsNoFiles(session, new Path(location));
-            external = false;
+        if (location != null) {
+            throw new TrinoException(
+                    NOT_SUPPORTED,
+                    "Using CREATE TABLE with an existing table content is deprecated, instead use the system.register_table() procedure." +
+                            " The CREATE TABLE syntax can be temporarily re-enabled using the 'delta.legacy-create-table-with-existing-location.enabled' config property" +
```

Comment:
This PR removes "delta.legacy-create-table-with-existing-location.enable" session property. I don't think users can re-enable it.